pdns/rec_channel_rec.cc

```diff
@@ -928,6 +928,27 @@ static uint64_t doGetThreadCPUMsec(int n)
   return tt.times.at(n);
 }
 
+static ProxyMappingStats_t* pleaseGetProxyMappingStats()
+{
+  auto ret = new ProxyMappingStats_t;
```

Comment:
Why does this function return a raw pointer instead of returning, say, an owned object or a unique_ptr?

Reply:
This is old code, passing (pointers to) objects between threads. One day we will fix that, but now is not the right time. Remi already made a start with a better approach, but it is not finished yet, see #11427.